strategy.md

```diff
@@ -0,0 +1,58 @@
+# Strategy Pattern
+
+## Definition
+Das Strategypattern ermöglicht es, den Algorithmus unabhängig von ihn nutzenden Klienten zu variieren.
```

Comment:
bitte leerzeile zwischen der überschrift und den textblöcken einfügen. Ein markdown validator ist in der aktuellen README.md verlinkt.